.zuul.yaml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: dnf-run-tests
+    description: Run DNF tests
+    roles:
+      - zuul: zuul-distro-jobs
+    requires: repo
+#    require-projects:
+#      - rpm-software-management/ci-dnf-stack
+    branches:
+      - master
+    vars:
```

Comment:
The vars that will be exposed to the job playbook via the job ansible inventory (line 22 of the inventory)